.eslintrc.js

```diff
@@ -31,6 +31,18 @@ module.exports = {
       rules: {
         '@typescript-eslint/no-explicit-any': 'off'
       }
+    },
+    // Linting for tests
+    {
+      files: [
+        'packages/core/**/*.test.[tj]s?(x)'
```

Comment:
Linting coverage will be expanded as we roll out jest. I separated the test fixes from the lint fixes in the commit history. If you look at the individually you can see the actual test changes required to make the tests work are minimal (in this case). Fixing lint errors and warnings creates a bigger diff but it mostly around using the safer return Promise approach for async tests, which is auto fixable.